Build failure thanks to web ui merge

I get this as a result of the recent web ui merge in #6344.

```
   [607/616] COFFEE build/drivers/javascript/coffee/util.js
    [608/616] CP ./build/packages/js
    [609/616] CP build/packages/js/node_modules/bluebird
    [610/616] BROWSERIFY build/drivers/javascript/rethinkdb.js
    [611/616] CP build/web_assets/rethinkdb.js
    [612/616] COFFEE build/web_assets/cluster.js
    [613/616] LESSC build/web_assets/cluster.css
/usr/bin/env: node: No such file or directory

Full error log: ./build/web_assets_js.log
admin/build.mk:27: recipe for target 'build/web_assets/cluster.js' failed
make[1]: *** [build/web_assets/cluster.js] Error 1
/usr/bin/env: node: No such file or directory

Full error log: ./build/web_assets_css.log
admin/build.mk:31: recipe for target 'build/web_assets/cluster.css' failed
make[1]: *** [build/web_assets/cluster.css] Error 1
make[1]: Target 'default-goal' not remade because of errors.
Makefile:52: recipe for target 'make' failed
make: *** [make] Error 2
```

On Debian Jessie with `./configure --allow-fetch CXX=clang++`

--- a/mk/support/pkg/node.sh
+++ b/mk/support/pkg/node.sh
@@ -2,3 +2,15 @@ version=6.10.0
 
 src_url=http://nodejs.org/dist/v$version/node-v$version.tar.gz
 src_url_sha1=3bb2629ed623f38b8c3011cf422333862d3653a3
+
+pkg_install () {
+    pkg_copy_src_to_build
+    pkg_configure
+    pkg_make install
+
+    mv "$install_dir"/bin/npm "$install_dir"/bin/npm.orig
+    { echo "#!/bin/sh"
+      echo "exec $install_dir/bin/node $install_dir/bin/npm.orig "'"$@"'
+    } > "$install_dir"/bin/npm
+    chmod +x "$install_dir"/bin/npm
+}

